commands.lua

```diff
@@ -61,7 +129,7 @@ function RFP.SET_REMOTE_SENSOR(idBinding, strCommand, tParams)
 end
 
 function RFP.VALUE_INITIALIZE(idBinding, strCommand, tParams)
-    RFP:VALUE_INITIALIZED(strCommand, tParams)
+    RFP.VALUE_INITIALIZED(strCommand, tParams)
```

Comment:
If we change this to a dot operator, we'll need to make sure we pass the first param (idBinding)